lib/ansible/modules/network/nxos/nxos_logging.py

```diff
@@ -286,6 +285,30 @@ def map_config_to_obj(module):
                         'dest_level': parse_dest_level(line, dest, parse_name(line, dest)),
                         'facility_level': parse_facility_level(line, facility)})
 
+    cmd_1 = [{'command': 'show logging | section enabled | section console', 'output': 'text'}]
```

Comment:
Why not just make this one call?

e.g.:

```
cmds = [{'command': <...>}, {'command': <...>}]
default_data = run_commands(module, cmds)
```

Reply:
Yes, I tried that @Qalthos but, it is returning the data in a format which would take much more logic to parse. Not sure if that much more lines of code is necessary just to handle two cases.